File: src/main/webapp/js/service/bookService.js
//服务层
app.service('bookService',function($http){
	
	 //预约
	this.add=function(entity){
		return $http.post('../book/add.do',entity);
	};

	//更改预约状态
	this.updateStatus=function(ids,status){
		return $http.get('../book/updateStatus.do?ids='+ids+"&status="+status);
	};

	//获取预约记录
	this.getByUsername=function(){
		return $http.get('../book/getByUsername.do');
	};

	//查找预约
	this.search=function(page,rows,entity){
		return $http.post('../book/search.do?page='+page+'&rows='+rows,entity);
	};

	//查看预约详情
	this.findOne=function(id){
		return $http.get('../book/findOne.do?id='+id);
	}
	

		
});

File: src/main/java/com/manicure/service/BookService.java
package com.manicure.service;

import com.manicure.entity.TbService;
import entity.PageResult;

import java.util.List;

public interface BookService {

    /**
     * 预约服务
     * @param service
     */
    public void add(TbService service);

    /**
     * 获取用胡预约记录
     * @param username
     * @return
     */
    public List<TbService> getByUsername(String username);

    /**
     * 后台获取预约记录/帅选
     * @param service
     * @return
     */
    public PageResult findPage(TbService service,int pageNum, int pageSize);

    /**
     * 管理员更新预约状态
     * @param ids
     * @param status
     */
    public void updateStatus(Long[] ids, String status);


    /**
     * 查看订单详情
     * @param id
     * @return
     */
    public TbService finfOne(Long id);

}
